.travis.yml

```diff
@@ -2,13 +2,20 @@ language: bash
 services: docker
 
 env:
-  image=eclecticiq/package
-
+  global:
+    image=eclecticiq/package
+    latest=1.11.0
+  matrix:
+    - version=1.11.0
+    - version=1.10.2
+    - version=1.10.1
+    - version=1.10.0
 script:
-  - travis_retry docker build --tag "$image:latest" .
+  - travis_retry docker build --build-arg VERSION=$version --tag "$image:$version" .
 
 after_success:
   - if [ "$TRAVIS_BRANCH" == "master" ]; then
+    docker tag "$image:$latest" "$image:latest"
```

Comment:
won't this fail when `version=0.10.2` but `latest=1.11.0` because `$image:$latest` won't be built on that particular travis worker? maybe this needs an `if [ "$version" = "$latest" ]` ?

Reply:
Good point, I can use an `if` or build stages for that.